File: src/test/java/no/nav/fo/veilarbregistrering/bruker/FoedselsnummerTest.java
package no.nav.fo.veilarbregistrering.bruker;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class FoedselsnummerTest {

    @Test
    public void maskert_skal_maskere_alle_tegn_med_stjerne() {
        Foedselsnummer foedselsnummer = Foedselsnummer.of("23067822521");
        assertThat(foedselsnummer.maskert()).isEqualTo("***********");
    }
}

File: src/test/java/no/nav/fo/veilarbregistrering/bruker/FoedselsnummerTest.kt
package no.nav.fo.veilarbregistrering.bruker

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class FoedselsnummerTest {

    @Test
    fun `maskert skal maskere alle tegn med stjerne`() {
        val foedselsnummer = Foedselsnummer.of("23067822521")
        assertThat(foedselsnummer.maskert()).isEqualTo("***********")
    }
}
